File: Add Binary/Add_Binary.py
class Solution:
    def addBinary(self, a: str, b: str) -> str:
        i = len(a) - 1
        j = len(b) - 1
        sodu = 0
        ketqua=[]
        while i >= 0 or j >= 0 or sodu:
            tong = sodu
            if i>= 0:
                tong += int(a[i])
                i -= 1
            if j>= 0:
                tong += int(b[j])
                j -= 1
            ketqua.append(str(tong % 2))
            sodu = tong // 2
        return "".join(reversed(ketqua))
        
a = "11"
b = '1'
s=Solution()
print(s.addBinary(a = a,b = b))

File: Add Binary/AddBinary.js
/**
 * @param {string} a
 * @param {string} b
 * @return {string}
 */
var addBinary = function(a, b) {
    var i = a.length - 1;
    var j = b.length - 1;
    var sodu = 0;
    var result = [];
    function check (a) {
        if (a == 0) return false;
        return true;
    }
    while ( i>= 0 || j >= 0 || check(sodu)) {
        var tong = sodu;
        if (i >= 0) {
            tong += Number(a[i]);
            i--;
        }
        if (j >= 0) {
            tong += Number(b[j]);
            j--;
        }
        result.push(tong % 2);
        sodu = Math.floor(tong / 2);

    }
    return result.reverse().join("");
};